### PR TITLE
- Fix a bug where combo box can't be closed by clicking the header wh…

### PR DESCRIPTION
**Pull request by kamil-zubair**
…en NK_BUTTON_TRIGGER_ON_RELEASE is defined

### CODE REVIEW

**Comment by kamil-zubair** [comment]
Done, let me know if there is anything else that you need.

File: src/nuklear_popup.c [hunk 1]
```diff
@@ -134,7 +134,11 @@ nk_nonblock_begin(struct nk_context *ctx,
     } else {
         /* close the popup if user pressed outside or in the header */
         int pressed, in_body, in_header;
+#ifdef NK_BUTTON_TRIGGER_ON_RELEASE
+		pressed = nk_input_is_mouse_released(&ctx->input, NK_BUTTON_LEFT);
```
Comment (dumblob) [inline src/nuklear_popup.c]: Feel free to correct the indentation (we're using spaces) :wink:.

Comment (kamil-zubair) [reply src/nuklear_popup.c]: Done

**Comment by dumblob** [comment]
Oops, right now I noticed that the amalgamation didn't work quite right for some reason - many `LF`s got changed to `CRLF`s or even just `CR`s. In this state I can't merge it - we need just `LF`s.

File: nuklear.h [hunk 2]
```diff
@@ -16922,7 +16923,11 @@ nk_nonblock_begin(struct nk_context *ctx,
     } else {
         /* close the popup if user pressed outside or in the header */
         int pressed, in_body, in_header;
+#ifdef NK_BUTTON_TRIGGER_ON_RELEASE
+		pressed = nk_input_is_mouse_released(&ctx->input, NK_BUTTON_LEFT);
```
Comment (dumblob) [inline nuklear.h]: That's weird - here I see tabs, but in `nuklear_popup.c` there are spaces :confused:.

**Comment by dumblob** [comment]
Closing in favor of https://github.com/vurtun/nuklear/pull/908 . Thanks for the fix @kamil-zubair !

**Pull request closed without merging**